src/main/kotlin/app/batch/S3DirectoryReader.kt

```diff
@@ -62,6 +62,10 @@ class S3DirectoryReader(private val s3Client: AmazonS3, private val s3Utils: S3U
                 request.continuationToken = results.nextContinuationToken
             } while (results != null && results.isTruncated)
 
+            logger.info("s3 object count",
+                    "bucket_name" to bucketName,
+                    "s3_prefix_folder" to s3Utils.s3PrefixFolder, "object_count" to objectSummaries.count().toString())
```

Comment:
Do we have topic name here or is it added to every single log message anyway? Would be good to log it here if not.

Reply:
It's in every message.